### PR TITLE
Tell people they can't upload these small ass shards

### PR DESCRIPTION
**Pull request by aleitner**
Otherwise they could attack us with them

### CODE REVIEW

**Review by dylanlott — APPROVED** [review]
Looks good.

**Comment by phutchins** [comment]
LGTM.

File: lib/models/frame.js [hunk 1]
```diff
@@ -106,6 +106,25 @@ Frame.methods.addShard = function(pointer, callback) {
...
+          return false;
+        }
+      }
+    }
+
+    return true;
+  }
+
+  if (!shouldAddShard()) {
+    return callback(new Error('Shard size is too small'));
```
Comment (braydonf) [inline lib/models/frame.js:125]: We should do a bad request error here.

**Comment by braydonf** [comment]
Before we release this, I'm getting these errors when shard size should be okay:

```
0|bridge   | {"level":"error","message":"request error: Shard size is too small","timestamp":"2017-06-02T21:49:46.390Z"}
0|bridge   | {"level":"error","message":"undefined","timestamp":"2017-06-02T21:49:46.391Z"}
```

**Comment by braydonf** [comment]
To summarize why it's not working. The last data shard is not the last shard as there are parity shards that follow it. So excluding the last shard isn't excluding the last data shard.

**Comment by braydonf** [comment]
Fixed in https://github.com/Storj/service-storage-models/pull/109

**Pull request merged**